src/components/Geometry.vue

```diff
@@ -104,9 +104,11 @@ export default {
       this.vtk.renderWindow.render();
     },
     resize() {
-      if (this.vtk && this.vtk.renderWindowContainer) {
-        this.vtk.renderWindowContainer.resize();
-      }
+      this.$nextTick().then(() => {
```

Comment:
You could just `await this.$nextTick();`. I frequently end up injecting that idiom (more frequently than I'd like) in places like this.

You'd need to make the function `async`, but we don't care about the return value, so that's fine.